language/solana/move-to-solana/src/stackless/dwarf.rs

```diff
@@ -554,10 +732,39 @@ impl<'up> DIBuilder<'up> {
             let name_cstr = unsafe { std::ffi::CStr::from_ptr(name_ptr) };
             name_cstr.to_string_lossy().into_owned()
         } else {
-            "unknown name".to_string()
+            "unknown_name".to_string()
+        }
+    }
+
+    pub fn get_operand_name(instruction: LLVMValueRef, idx: u32) -> String {
+        let operand_value = unsafe { LLVMGetOperand(instruction, idx) }; // operands are counted from 0
+        if !operand_value.is_null() {
+            let mut length: ::libc::size_t = 0;
+            let name_ptr = unsafe { LLVMGetValueName2(operand_value, &mut length) };
+            let name_cstr = unsafe { std::ffi::CStr::from_ptr(name_ptr) };
+            name_cstr.to_string_lossy().into_owned()
+        } else {
+            "unknown_name".to_string()
```

Comment:
No. This is an auxiliary function and "unknown_name" is in the else-clause (I do not know if we ever hit it).
If we need more tests it will be another PR.
The core of this PR is in finding a "magic" combination of parameters (described above in "Notable changes") that makes module verification happy.
I want to land this PR to the repo before I do any additional changes so we would have a working baseline.